src/seedu/addressbook/commands/SortCommand.java

```diff
@@ -0,0 +1,27 @@
+package seedu.addressbook.commands;
+
+import java.util.List;
+
+import seedu.addressbook.data.person.ReadOnlyPerson;
+
+
+/**
+ * Sort all persons in the address book in lexicographical order.
+ */
+public class SortCommand extends Command {
```

Comment:
Perhaps you can mention that this command sorts in a *case-insensitive* lexicographical order?